packages/unmock-core/src/nock.ts

```diff
@@ -166,58 +170,75 @@ export class DynamicServiceSpec {
     } else {
       this.data = JSONSchemify(maybeStatusCode) as Schema;
     }
-    return this.updater({ data: this.data, statusCode: this.statusCode });
+    const { store, service } = this.updater({
+      data: this.data,
+      statusCode: this.statusCode,
+    });
+
+    return buildFluentNock(store, this.baseUrl, this.name, service);
   }
 }
 
-export const nockify = ({
-  backend,
-  baseUrl,
-  name,
-}: {
-  backend: NodeBackend;
-  baseUrl: string;
-  name?: string;
-}) => {
+type FluentDyamicService = {
+  [k in HTTPMethod]: (endpoint: string) => DynamicServiceSpec;
+} & { state: StateType; spy: ServiceSpy };
```

Comment:
Should this have all the methods of `Service` or `IService` (whichever we expose) instead of hard-coding the properties it should have?

Reply:
Like does it have `reset`?

Reply:
Yup, re my conversation with Mike; will remove these anyway so it'll be more like
```
const dynamicServiceSpec = unmock.nock(....).get(...).reply(...).post(...).reply(...)
const service = unmock.services[...]
```
That will avoid the confusion.